src/backend/src/services/change-requests.services.ts

```diff
@@ -341,13 +342,22 @@ export default class ChangeRequestsService {
           projectNumber,
           workPackageNumber
         }
+      },
+      include: {
+        changeRequests: true
       }
     });
 
     if (!wbsElement) throw new NotFoundException('WBS Element', wbsPipe({ carNumber, projectNumber, workPackageNumber }));
     if (wbsElement.dateDeleted)
       throw new DeletedException('WBS Element', wbsPipe({ carNumber, projectNumber, workPackageNumber }));
 
+    const { changeRequests } = wbsElement;
+    changeRequests.filter((changeRequest) => !changeRequest.dateDeleted);
```

Comment:
filter doesn't actually mutate the original array, it just returns an updated array, so you'll need to make a new variable and then call that in the if statement